.github/workflows/gradle-ci.yml

```diff
@@ -29,9 +29,20 @@ jobs:
     - name: Build with Gradle
       run: ./gradlew build
 
-    - name: Publish to GitHub Packages
+    - name: Build OCI image and publish it to GitHub Packages
       if: github.ref == 'refs/heads/main'
       run: ./gradlew bootBuildImage --publishImage
       env:
         GITHUB_ACTOR: ${{ github.actor }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build OCI image
+      if: github.ref != 'refs/heads/main'
+      run: ./gradlew bootBuildImage
+      env:
+        GITHUB_ACTOR: ${{ github.actor }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run Cucumber
```

Comment:
welke versie van de applicatie wordt hier getest? de versie die hierboven gebouwd is of?

Reply:
de versie in de branch waarop de PR is gemaakt